multinode-demo/clear-config.sh

```diff
@@ -11,9 +11,7 @@ set -e
 
 (
   set -x
-  rm -rf "${SOLANA_CONFIG_DIR:?}/" # <-- $i might be a symlink, rm the other side of it first
-  rm -rf "$SOLANA_CONFIG_DIR"
-  mkdir -p "$SOLANA_CONFIG_DIR"
+  setup_secondary_mount
+  echo "Removing config dir"
+  rm -rf ${SOLANA_CONFIG_DIR}/*
```

Comment:
Probably want to make this `rm -rf "${SOLANA_CONFIG_DIR:?}"/*`.  Otherwise BadThings(TM) might happen if `SOLANA_CONFIG_DIR` isn't set